.github/workflows/go.yml

```diff
@@ -210,20 +235,28 @@ jobs:
         with:
           name: k0s
 
-      - name: Restore airgap bundle
-        uses: actions/cache@v3
-        with:
-          key: ${{ runner.os }}-build-airgap-bundle-${{ github.ref_name }}-${{ github.sha }}
-          path: |
-            image-bundle/bundle.tar
-
       - name: k0s sysinfo
         run: |
           chmod +x k0s
           ./k0s sysinfo
 
+      - name: Cache airgap image bundle
+        id: cache-airgap-image-bundle
+        if: contains(matrix.smoke-suite, 'airgap')
+        uses: actions/cache@v3
+        with:
+          key: airgap-image-bundle-linux-amd64.tar-${{ needs.build.outputs.airgap-image-bundle-hash-linux }}
+          path: airgap-image-bundle-linux-amd64.tar
```

Comment:
Can't we save the bundle under the `image-bundle/bundle.tar` path and keep the other code as is?

Reply:
I personally think this change makes the code easier and clearer to follow by depicting exactly which image bundle we are using.